src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -144,6 +145,14 @@ export default function SnapshotListModal() {
 
   const loadingOrZeroCount = fetchingOrLoading || !count;
 
+  const onPackageCountClick = (snapUUID) => {
+    const SnapshotListModalRoute = `/content/${uuid}/snapshots/${snapUUID}/packages`;
+
+    navigate(
+      `${window.location.pathname}?modalRoute=${encodeURIComponent(SnapshotListModalRoute)}`,
```

Comment:
not sure where you found this, but we have [a different pattern](https://github.com/Andrewgdewar/content-sources-frontend/blob/fe21d61edd93c360cc74026642aa8ba58baa6ad5/src/Pages/AdminTaskTable/components/ViewPayloadModal/ViewPayloadModal.tsx#L40) to use for routing.

Reply:
Look at the linked pattern above, use a combination of that and the snapshotlistmodalroute youv'e created above to get the navigate opening your modal.

Reply:
Hint: [This](https://github.com/content-services/content-sources-frontend/pull/207/files#diff-21c0871aec2cc959996fa3600d513fccb3575d5d72dd892b0102df47b62058acR45) is the route that you are trying to go to.

Reply:
Hint: [This](https://github.com/content-services/content-sources-frontend/pull/207/files#diff-21c0871aec2cc959996fa3600d513fccb3575d5d72dd892b0102df47b62058acR44) is where you are when you click the link.